src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveLoadVectorFirstFaultingTest.template

```diff
@@ -144,10 +144,11 @@ namespace JIT.HardwareIntrinsics.Arm
         private static readonly int LargestVectorSize = {LargestVectorSize};
 
         private static readonly int RetElementCount = Unsafe.SizeOf<{RetVectorType}<{RetBaseType}>>() / sizeof({RetBaseType});
+        private static readonly int Op1ElementCount = Unsafe.SizeOf<{Op1VectorType}<{Op1BaseType}>>() / sizeof({Op1BaseType});
```

Comment:
i was hoping to see similar validation in other *FirstFaulting*.template.

Reply:
The other two `FirstFaulting` templates we have at the moment are `SveGatherVectorFirstFaultingVectorBases.template` and `SveGatherVectorFirstFaultingIndices.template`. Both have a similar validation already. Please check usage of the following variables. 
https://github.com/dotnet/runtime/blob/6169e41a6ba28843d18a778a4e5e0edebed54414/src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveGatherVectorFirstFaultingVectorBases.template#L308
https://github.com/dotnet/runtime/blob/6169e41a6ba28843d18a778a4e5e0edebed54414/src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveGatherVectorFirstFaultingIndices.template#L285

You may also note that the `ValidateFirstFaultingResult` methods accept load masks as first parameter:
https://github.com/dotnet/runtime/blob/6169e41a6ba28843d18a778a4e5e0edebed54414/src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveGatherVectorFirstFaultingVectorBases.template#L665
https://github.com/dotnet/runtime/blob/6169e41a6ba28843d18a778a4e5e0edebed54414/src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveGatherVectorFirstFaultingIndices.template#L644